src/components/AppSidebar.vue

```diff
@@ -6,12 +6,11 @@
 
 aside {
   background: white;
-  height: 100px;
+  height: 86px;
   transition: height 0.2s ease-in-out;
 
   @media (min-width: $breakpoint-tablet-portrait) {
-    width: 244px;
-    padding: 24px 48px;
+    width: 340px;
     height: unset;
```

Comment:
Does the CSS postprocessor pick this up?? The `unset` value is not supported in the IEs: https://caniuse.com/#search=unset

Reply:
The CLI does have a post-processor that sweeps the code, I'm just not very educated on it, so I have no idea. I'm researching this now, good that you brought it up!

Reply:
Mkay, so I just checked, the post processor does NOT pick this up. I am looking into fixing this.

Reply:
I looked into this a bit, the best alternative ATM is to ditch usage of "unset".Or do you know of any postcss plugin that might solve this?

Reply:
No, not aware of any. Sorry.

Reply:
Alright, I removed the property all together and will refrain from using it. I was also using initial, turns out that it is also not supported by IE... Pity!